relengapi/blueprints/test_slaveloan.py

```diff
@@ -14,4 +12,3 @@
 def test_hello(client):
     rv = client.get('/slaveloan/')
     eq_(rv.status_code, 200)
-    eq_(json.loads(rv.data)['result'], {'message': 'hello world'})
```

Comment:
This absolutely has to have *some* tests, even if those don't push it to 100% coverage.  Without tests, the code is inapproachable by anyone but you, since the rest of us don't know how to actually verify that a change has not broken something important.